resources/test/fixtures/pyupgrade/UP022.py

```diff
@@ -0,0 +1,17 @@
+import subprocess
+
+output = subprocess.run(["foo"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+output = subprocess.run(
+    ["foo"], stdout=subprocess.PIPE, check=True, stderr=subprocess.PIPE
+)
+
+output = subprocess.run(
+    ["foo"],
+    stdout=subprocess.PIPE,
+    check=True,
+    stderr=subprocess.PIPE,
```

Comment:
Yeah this is a bit tricky, since the kwargs can be non-consecutive, and we need to fix it with a single edit operation (one `::replacement` call).

I think the way to approach it would be:

- Find the first kwarg.
- Find the second kwarg.
- Find all content in-between (use `SourceCodeLocator` to slice the range between the two kwargs).
- Treat the fix as: replace (start of first kwarg through to end of second kwarg) with (`capture_output=True` + content between the two kwargs).

Reply:
Perfect, I like this solution.

Reply:
I figured it out using your solution, but we could just make a bunch of fix functions. Like replace_2_lines, or replace_1_delete_1. But for this specific case. I think what we have now is best.